platform/darwin/src/MGLRendererConfiguration.h

```diff
@@ -35,6 +35,19 @@ NS_ASSUME_NONNULL_BEGIN
  which will be available at run time, e.g. "PingFang". */
 @property (nonatomic, readonly) mbgl::optional<std::string> localFontFamilyName;
 
+/**
+ A Boolean value indicating whether symbol layers may enable cross-source symbol
+ collision detection.
+ 
+ Set `MGLCrossSourceCollisions` in your containing app's Info.plist
```

Comment:
Since the header is internal, this documentation isn’t visible to developers. Add an entry about the Info.plist key to the “Info.plist Keys” guide.